<launch>

	<arg name="namespace" default="Sauron"/>

    <node pkg="batch_pose_estimator" type="batch_solver" name="batch_solver" output="screen">

    	<!-- Minimum number of elements to start the batch solver -->
    	<param name="min_elements" value="200"/>

		<!-- Image topics -->
		<!-- NOTE: input topics have to be time-synchronized (header/stamp should match) -->
		<param name="input_pose_topic" value="/t265/odom/center_of_mass"/>
		<param name="input_slam_topic" value="/RGBD/COMPoseENUFrame"/>

		<!-- Namespaces of input topics -->
		<rosparam param="namespaces" subst_value="true"> ["Sauron", "Bilbo"] </rosparam>
		

		<!-- Camera position in body frame (in meters) -->
		<rosparam param="cam_pos_body_frame"> [0.0, 0.0, 0.0] </rosparam>
	</node>

</launch>